### PR TITLE
Added mood tiles

### PR DESCRIPTION
**Pull request by jiyasaraf**
I have added 5 mood tiles in the mood tiles section.

### CODE REVIEW

File: app/src/main/java/live/groupstudy/gs/activities/login/LoginActivity.kt [hunk 1]
```diff
@@ -48,6 +48,7 @@ class LoginActivity : ComponentActivity() {
     companion object {
         const val PHONE_NUMBER = "phoneNumber"
         const val TAG = "LoginActivity"
+        const val trial="test case"
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/login/LoginActivity.kt]: Remove this as it was only for testing.

File: app/src/main/java/live/groupstudy/gs/activities/mood/MoodActivity.kt [hunk 2]
```diff
@@ -23,6 +44,7 @@ class MoodActivity : ComponentActivity() {
                     color = MaterialTheme.colorScheme.background
                 ) {
                     Greeting("Android")
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/mood/MoodActivity.kt]: Greeting is not reequired.

File: app/src/main/java/live/groupstudy/gs/activities/mood/MoodActivity.kt [hunk 3]
```diff
@@ -41,6 +63,146 @@ fun Greeting(name: String, modifier: Modifier = Modifier) {
 @Composable
 fun GreetingPreview() {
     GroupStudyTheme {
-        Greeting("Android")
+//        Greeting("Android")
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/mood/MoodActivity.kt]: remove this line of code.

File: app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt:1]: Optimize the code. Use Lazy column.

File: app/src/main/java/live/groupstudy/gs/repositories/LoginRepository.kt [hunk 4]
```diff
@@ -3,8 +3,8 @@ package live.groupstudy.gs.repositories
 import live.groupstudy.gs.datasources.FirebaseDataSource
 
 class LoginRepository(private val firebaseDataSource: FirebaseDataSource) {
-
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/repositories/LoginRepository.kt:6]: only push your changes. Don't add any extra lines.

File: app/src/main/res/drawable/chanakya.png
Comment (ranjanproject) [inline app/src/main/res/drawable/chanakya.png:1]: use extracted images not clipped one.

File: app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt [hunk 5]
```diff
@@ -22,73 +29,83 @@ import androidx.compose.runtime.remember
...
+                    )
+                LazyVerticalGrid(
+                    columns = GridCells.Fixed(2),
+                    contentPadding = PaddingValues(40.dp),
+                    verticalArrangement = Arrangement.spacedBy(20.dp),
+                    horizontalArrangement = Arrangement.spacedBy(20.dp)
+                ) {
+                    items(moodItemList) { it ->
+                        MoodTile(it)
+                        Spacer(modifier = Modifier.height(150.dp));
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt]: this code is not in correct format.

File: app/src/main/java/live/groupstudy/gs/activities/mood/TotalCountSilentBuddha.kt [hunk 6]
```diff
@@ -0,0 +1,47 @@
+package live.groupstudy.gs.activities.mood
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/mood/TotalCountSilentBuddha.kt]: why this code is present in this branch? I have already told you to create another branch for other task.

File: app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt [hunk 7]
```diff
@@ -22,73 +29,83 @@ import androidx.compose.runtime.remember
...
     ) {
-
         Image(
-            painter = painterResource(id = R.drawable.ic_launcher_foreground),
+            painter = painterResource(id = moodTileItem.imageId),
             contentDescription = null,
             modifier = Modifier
-                .fillMaxWidth()
-                .height(dimensionResource(id = R.dimen.dimen_85))
+                .width(152.dp)
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt]: why you have used hardcoded value? add these values in dimen file.

File: app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt (hunk 5 above)
Comment (jiyasaraf) [reply app/src/main/java/live/groupstudy/gs/activities/mood/MoodTile.kt]: What should I change to make it in the correct format?
That spacer line, I have removed. Any other changes required for this snippet?

File: app/src/main/java/live/groupstudy/gs/repositories/LoginRepository.kt [hunk 8]
```diff
@@ -3,8 +3,8 @@ package live.groupstudy.gs.repositories
...
     fun verifyNumber(number: String){
         firebaseDataSource.verifyNumber(number)
+
```
Comment (ranjanproject) [inline app/src/main/java/live/groupstudy/gs/repositories/LoginRepository.kt]: remove this line.

**Pull request closed without merging**